ourpath.py

```diff
@@ -0,0 +1,83 @@
+import osmnx as ox
+import networkx as nx
+from shapely.geometry import Polygon
+import matplotlib.pyplot as plt
+import folium
+
+    # Function to save plot as image
+def plot_graph(G):
+    fig, ax = ox.plot_graph(G, show=False, close=False)
+    return fig, ax
+    
+    
+def get_path(want_graph):
+
+    ## Using boundbox
+    # cmrit lat long bounds 
+    # north (float) – northern latitude of bounding box: 12.96768338736144
+    # south (float) – southern latitude of bounding box: 12.965518842636625
+    # east (float) – eastern longitude of bounding box: 77.71393642911939
+    # west (float) – western longitude of bounding box: 77.71046157295785
+    
+    
+    north, south, east, west = 12.96768338736144,12.965518842636625,77.71393642911939,77.71046157295785
+    network_type = 'all_private' # "all_private", "all", "bike", "drive", "drive_service", "walk"
+
+    G = ox.graph_from_bbox(
+        north, south, east, west, 
+        network_type=network_type, 
+        simplify=True, 
+        retain_all=False, 
+        truncate_by_edge=False, 
+        clean_periphery=True, 
+        custom_filter=None)
+
+    # Renaming the existing nodes in the graph
+    mapping = {3798918923:'cmrit_entrance',4159727902:'teacher_parking',4159727907:'volley_ball_court' }
```

Comment:
Am thinking of creating a class for CMRIT path as of now. 
so in init ...we could have the mapping, node+edge additions done.
and then the other methods defined  to the class.

A general way of addressing this problem as a Class would require more work and understanding of our application in the fields.

So I believe the proposed method mentioned in the beginning would be right to go ahead with.

Reply:
Why is that so? We can add and modify methods as and when required right?

Reply:
yes we can. I am talking AS OF NOW. Our focus is cmrit field.  

Obviously we cant use the current code for a field that may have around 20 nodes , for that we need high modification of the existing code as to how easily we can add nodes and edges  (mapping) without having to enter them manually 1 by 1. 
That's what we r doing with CMRIT Pathway right.